gcam_to_tell/README.MD

```diff
@@ -25,41 +26,26 @@ TELL needs the following information:
 
 | Source | Description | Output Variable Name | GCAM Query/File Path(s) | Original Units | Output Units |
 |:-:|:-:|:-:|:-:|:-:|:-:|
-| GCAM Output | Annual Total Electricity Consumption by State and Sector | energyFinalConsumBySecEJ | "TBD" | EJ | EJ |
-
-
-### Notes
-
-* TBD
+| GCAM Output | Annual Total Electricity Demand by State | energyFinalConsumBySecEJ | "TBD" | EJ | EJ |
 
+## Notes
+* The GCAM output database has total annual state-level electricity demand by sector. Either GCAM-reader or the 
+associated python script should sum the sector-specific values to create a single total annual state-level electricity 
+demand value for each year.
+* We should double check the units. GCAM seems to simulate demand in EJ, but the output of the GCAM reader is in TWh. We

```

Comment:
GCAM and therefore GCAM reader would only operate in EJ, never TWh.  Maybe this was an artifact from using `gcam_extractor` which would do the unit conversion?